accessibility-scan.yml

```diff
@@ -0,0 +1,25 @@
+jobs:
+  - name: accessibility-scan
+    plan:
+    - task: run-pa11y
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: node
+        run:
+          path: sh
+          args:
+          - -exc
+          - |
+            npm install -g pa11y-crawl
```

Comment:
`npm install [package_name]@[version]` adding this here for reference if we ever need it.